packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -243,9 +243,10 @@ export class FormCreationService {
 
   private setDefaultValues(form: Form, createFormParams: CreateFormParams) {
     const { session } = createFormParams;
-
+    const currentVisitIsRetrospective = this.singleSpaPropsService.getPropOrThrow('currentVisitIsRetrospective');
+    const visitStartDatetime = this.singleSpaPropsService.getPropOrThrow('visitStartDatetime');
```

Comment:
Seems wrong to require this property if we only use it when a different property is set...